src/main/java/pl/bykowsi/kurs/tydzien1pd/screeninfo/MessagesPrinter.java

```diff
@@ -0,0 +1,56 @@
+package pl.bykowsi.kurs.tydzien1pd.screeninfo;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.bykowsi.kurs.tydzien1pd.configuration.LanguageSettings;
+import pl.bykowsi.kurs.tydzien1pd.model.PriceCalculationsData;
+import pl.bykowsi.kurs.tydzien1pd.model.Product;
+
+import java.util.List;
+import java.util.Locale;
+
+
+public class MessagesPrinter {
+
+    private static Logger logger = LoggerFactory.getLogger(MessagesPrinter.class);
+
+    public static void printBasket(List<Product> list, LanguageSettings languageSettings) {
+        logger.warn("");
+        list.forEach(p -> logger.warn(languageSettings.getMessageSource().getMessage("singleProductPosition", new Object[]{p.getName(), p.getPrice()}, Locale.forLanguageTag(languageSettings.getLanguageVersion()))));
```

Comment:
można by to:
`languageSettings.getMessageSource().getMessage("singleProductPosition", new Object[]{p.getName(), p.getPrice()}, Locale.forLanguageTag(languageSettings.getLanguageVersion())))`
 wydzielić to prywatnej metody

Reply:
i dodać entery

Reply:
niestety nie moge tak zrobić, wynika to z tego, że jako 2-gi argument metody getMessage() mam tablicę i w zależności którą linijke tekstu chcę wyświetlić to różnią się one ilością parametrów. Ta tablica to własnie są te parametry, a nie mam możliwości dostosowania rozmiaru tablicyprzy wydzieleniu do metody